src/GUI/EFCorePowerTools/Handlers/ReverseEngineerHandler.cs

```diff
@@ -148,10 +148,17 @@ public async System.Threading.Tasks.Task ReverseEngineerCodeFirstAsync(Project p
                     }
                 }
 
+                for (var i = 0; i < predefinedTables.Count; i++)
```

Comment:
Suggest you comment this code out with a TODO, and I will investigate once this is merged,

Reply:
If I comment this code, the reveng process would overwrite modifications to efpt.config.json file. Is it ok?

Reply:
Yes, I will do some testing when this has been merged